node_cli/utils/global_config.py

```diff
@@ -0,0 +1,66 @@
+#   -*- coding: utf-8 -*-
+#
+#   This file is part of node-cli
+#
+#   Copyright (C) 2021 SKALE Labs
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import os
+
+from node_cli.core.host import safe_mk_dirs
+from node_cli.utils.helper import write_json, read_json
+from node_cli.configs import GLOBAL_SKALE_CONF_FILEPATH, GLOBAL_SKALE_DIR
+
+
+def read_g_config() -> dict:
+    """Read global SKALE config file, init if not exists"""
+    try:
+        return read_json(GLOBAL_SKALE_CONF_FILEPATH)
+    except FileNotFoundError:
```

Comment:
I suggest to use if statement and  os.path.isfile instead of FileNotFoundError

Reply:
changed